content/telegraf/v1.21/about_the_project/release-notes-changelog.md

```diff
@@ -0,0 +1,3369 @@
+---
+title: Telegraf 1.20 release notes
+description: Important features and changes in the latest version of Telegraf.
+alias:
+  - /telegraf/v1.20/reference/release-notes/influxdb/
+menu:
+  telegraf_1_21:
+
+    name: Release notes
+    weight: 10
+    parent: About the project
+---
+## v1.21 [2021-12-15]
+
```

Comment:
can you make sure you add the note about RPM at the top? Maybe just

`The signing for RPM digest has changed to use `sha256` to improve security. Due to this change, RPM builds may not be compatible with RHEL6 and older releases (Telegraf only supports releases in RHEL production)`